notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
```

Comment:
I think these are only going to be appropriate for the length === 1 case. I don't think we can open all the selected files at once.